config/environments/development.rb

```diff
@@ -1,6 +1,7 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  ENV['EMAIL_DOMAIN'] = 'vibereport.com'
```

Comment:
Please put this at the bottom of the block, because we will have a lot of them after a while.